<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsDialog</class>
 <widget class="QDialog" name="SerialSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>AI Serial Port</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="selectBox">
        <property name="title">
         <string>Select Serial Port</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="serialPortInfoListBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="manufacturerLabel">
           <property name="text">
            <string>Manufacturer:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="vidLabel">
           <property name="text">
            <string>Vendor ID:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="pidLabel">
           <property name="text">
            <string>Product ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="updateBtn">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/rc.qrc">
             <normaloff>:/images/search_press.png</normaloff>:/images/search_press.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="serialNumberLabel">
           <property name="text">
            <string>Serial number:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parametersBox">
        <property name="title">
         <string>Select Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="baudRateLabel">
           <property name="text">
            <string>BaudRate:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="baudRateBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="dataBitsLabel">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="dataBitsBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="parityLabel">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="parityBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="stopBitsLabel">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="stopBitsBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="flowControlLabel">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="flowControlBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>AO Serial Port</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="selectBox_2">
        <property name="title">
         <string>Select Serial Port</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="AOdescriptionLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="AOserialPortInfoListBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="AOmanufacturerLabel">
           <property name="text">
            <string>Manufacturer:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="AOvidLabel">
           <property name="text">
            <string>Vendor ID:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="AOpidLabel">
           <property name="text">
            <string>Product ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="AOupdateBtn">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/rc.qrc">
             <normaloff>:/images/search_press.png</normaloff>:/images/search_press.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="AOserialNumberLabel">
           <property name="text">
            <string>Serial number:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parametersBox_2">
        <property name="title">
         <string>Select Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="AObaudRateLabel">
           <property name="text">
            <string>BaudRate:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="AObaudRateBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="AOdataBitsLabel">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="AOdataBitsBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="AOparityLabel">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="AOparityBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="AOstopBitsLabel">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="AOstopBitsBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="AOflowControlLabel">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="AOflowControlBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="disConnectBtn">
       <property name="text">
        <string>Disconnect Current I/O Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save COM Settomg</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/rc.qrc"/>
 </resources>
 <connections/>
</ui>
